src/amplitude.js

```diff
@@ -190,6 +190,44 @@ class Amplitude {
     // }
   }
 
+  /**
+   *  Disconnects the output of this p5.Amplitude object.
+   *
+   *  @method disconnect
+   *  @for p5.Amplitude
+   *  @example
+   *  <div><code>
+   *  let sound, amplitude;
+   *  function preload(){
+   *    sound = loadSound('assets/beat.mp3');
+   *  }
+   *
+   *  function setup() {
+   *    let cnv = createCanvas(200, 200);
+   *    cnv.mouseClicked(togglePlay);
+   *    amplitude = new p5.Amplitude(); 
+   *    sound.loop();
+   *  }
+   *
+   *  function draw() {
+   *    background(220);
+   *    text('Disconnect:', 20, 20);
```

Comment:
Maybe we can make the text "Click to disconnect" so it's clearer you can click anywhere?

Reply:
Ok, i do it.